cmake/thirdparty/get_gbench.cmake

```diff
@@ -9,13 +10,21 @@
 # is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
 # or implied. See the License for the specific language governing permissions and limitations under
 # the License.
+# =============================================================================
 
-# Generate version_config.hpp from the version found in CMakeLists.txt
```

Comment:
Very strange that this change shows up as a rename when the contents and directory are completely different.

Reply:
A side-effect of how git tracks file renames. Since a reasonable percentage of a new file is the same as the deleted file ( thanks to the copyright ) git decides that a mv + diff is more efficient.

I could separate out the delete and created into two separate commits, but I believe once the squash merge occurs we will get the mv + diff again in the git history.